src/backend/server.js

```diff
@@ -207,26 +201,36 @@ app.post("/admin/clear", async (req, res) => {
   }
 });
 
-
 app.post("/api/grid/:chatId/:userId", async (req, res) => {
   const chatId = req.params.chatId;
   const userId = req.params.userId;
-  const isPermitted = auth.authenticateChatId(chatId);
 
   const user = await db.getUserByTelegramId(userId)
+
+  // Verifies that the user exists in the database
+  if (user === null) {
```

Comment:
Consider using a try catch here instead of checking for null. If user isn’t in database, the query will throw an error that should be caught here

Reply:
Resolved.